testing/library/sandbox_runner/runner.go

```diff
@@ -57,15 +56,13 @@ func Run(cmd *exec.Cmd, opts ...SandboxOption) *gexec.Session {
 
 func WithFile(path string, contents []byte, envOpts ...EnvOption) SandboxOption {
 	return func(conf sandboxConfig) error {
-		filePath := filepath.Join(path)
-
-		err := ioutil.WriteFile(filePath, contents, os.FileMode(0644))
+		err := ioutil.WriteFile(path, contents, os.FileMode(0644))
```

Comment:
I guess `filepath.Join(path)` was made to remove trailing slash if it exists. (https://go.dev/play/p/YsAOH3XYOAO)
Maybe we should keep this behavior in a way like

```
path = strings.TrimSpace(path)
if strings.HasSuffix(path, "/") {
    path = strings.TrimSuffix(path, "/")
}
```